banner/svelte/src/App.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+  import Banner from './Banner.svelte';
+  import { onMount } from 'svelte';
+  import { ContentClient } from 'dc-delivery-sdk-js';
+
+  let banner = null;
+
+  const client = new ContentClient({
+    hubName: 'ampengineering',
+  });
+
+  const deliveryKey = 'banner-example';
+
+  onMount(async () => {
```

Comment:
Would be able to save a few lines by using a promise and an {#await} instead of onMount and await

Reply:
fixed, perhaps!